MozillaFoundation/foundation.mozilla.org
# Remove custom donate page footer

# Description

This PR removes the custom donate page footer. The footer should now inherit the default foundation footer.

Link to sample test page:
Related PRs/issues: #12021 TP1-33

# To test
1) Go to /donate and confirm the default foundation footer is loading
2) Go to /donate/help and confirm the default foundation footer is loading
